include/swift/Reflection/ReflectionContext.h

```diff
@@ -244,11 +215,13 @@ class ReflectionContext
         if (strncmp(S->sectname, Name.data(), strlen(Name.data())) != 0)
```

Comment:
Not your code, but it's inefficient to call `strlen(StringRef.data())`, this should be `Name.size()` instead.

Reply:
Perhaps you could fix that in a separate follow-up commit?

Reply:
Sure!